components/project/other/TruncatedAddressComponent.vue

```diff
@@ -23,5 +28,6 @@ const truncateAddress = (address: string) => {
 const copyAddress = (address: string) => {
   copy(address);
   message.info('Copied to clipboard');
+  (document.activeElement as HTMLElement | null)?.blur();
```

Comment:
I believe that `@click` can provide event information and from the event you can get target (active element). Using that over document.activeElement could be better approach.